pulp_container/app/registry.py

```diff
@@ -226,9 +295,103 @@ async def get_tag(self, request):
         # convert if necessary
         return await Registry.dispatch_converted_schema(tag, accepted_media_types, path)
 
+    async def save_manifest_and_blobs(
+        self, digest, manifest_data, media_type, remote, repository, artifact
+    ):
+        config_digest = manifest_data["config"]["digest"]
```

Comment:
how are you sure this is not a schema1 manifest?

Reply:
I am not. 😭